src/QuadForm/Herm/Mass.jl

```diff
@@ -149,7 +149,7 @@ function _local_factor_generic(L::HermLat, p)
   ss = elem_in_nf(uniformizer(p))^(-val)
 
   if def
-    R = base_ring(base_ring(L))
+    R = maximal_order(K)
```

Comment:
These two are not necessarily the same objects.

Reply:
I thought they were, like in the cases in which I tried to test this.. I could put it back, I just wanted to make the definition `R` "lighter".

Reply:
I would prefer keeping the previous version.